utility/baseNode.ts

```diff
@@ -0,0 +1,12 @@
+export const settledPromiseRejected = (result: PromiseSettledResult< unknown>) :
+																				result is PromiseRejectedResult => 
```

Comment:
Formatting please

Reply:
what's wrong with the formatting? seems clear to me?